web/react/components/edit_post_modal.jsx

```diff
@@ -72,6 +80,16 @@ export default class EditPostModal extends React.Component {
 
         $(ReactDOM.findDOMNode(this.refs.modal)).modal('show');
     }
+    handleKeyDown(e) {
+        if (this.state.ctrlSend === 'true' && e.keyCode === KeyCodes.ENTER && e.ctrlKey === true) {
+            this.handleEdit(e);
+        }
+    }
```

Comment:
I simplified it a little, so we dont have to check to if()s.
Other than that, sadly, both handleKeyDown and handleKeyPress are needed since handleKeyPress (for some reason?) does not register keypresses + ctrl key.